### ISSUE TITLE
bbb-conf --setsecret is not changing Security key

### ISSUE THREAD

**Issue by amishbhavsar** [bigbluebutton/bigbluebutton#16350]
Hi,
I am on BBB 2.6 Beta 4 and found that the command:
`bbb-conf --setsecret`
is not changing the security key.
Let me know what details you need to help you replicate the issue.

**Comment by hostbbb** [comment]
did you restart the services with bbb-conf --restart?

also are you using greenlight? the salt is stored in .env for greenlight, and for bbb web its in /etc/bigbluebluebutton, and the default place its installed..   should look at these 3 places to see what did not change.

**Comment by amishbhavsar** [comment]
Hi,
I re-ran the `bbb-conf --setsecret`
Then ran `bbb-conf --restart`
Checked in /etc/bigbluebutton/bbb-web.properties and also in /root/greenlight/.env
The Security key does not change anywhere.

**Comment by hostbbb** [comment]
/usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties

 

What about there?

**Comment by danielpetri1** [comment]
Can confirm that it did not change `/usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties` either.

**Comment by amishbhavsar** [comment]
So, here's what happened...
I found another set of instructions to change the secret here: https://docs.bigbluebutton.org/admin/customize.html#change-the-shared-secret
Following that, I changed the secret.
I checked /etc/bigbluebutton/bbb-web.properties - the new secret was updated here - yay!
I checked /root/greenlight/.env - the new secret was not updated here, so I manually updated it and replaced the .env file
I did not check /usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties, so I cannot comment on it.

But after doing the above, when I was trying to launch the class using Greenlight, I was getting the endpoint and URL error message. I double triple checked the .env file and still I kept getting the error. I got frustrated so, I simply nuked the server, and started a fresh installation of BBB 2.6 Beta 4. The new installation is done, so I am all set - at least for now.

I am happy to support testing this. Let me know if you want me to help.
